services/networking/loadbalancer/taxonomy.md

```diff
@@ -0,0 +1,75 @@
+| CCC-050329  | Request and response header transformations           | Can modify HTTP headers in both incoming requests and outgoing responses.                                                                                                                                                                                                                                                                     |
+| CCC-050330  | Retries                                               | Configure the conditions under which the load balancer retries failed requests, how long to wait before retrying, and the maximum number of retries permitted.                                                                                                                                                                                |
+| CCC-050331  | URL based routing                                     | Direct incoming requests to different backend resources based on the content of the request URL.                                                                                                                                                                                                                                              |
+| CCC-050332  | HTTP header based routing                             | Direct incoming requests to different backend resources based on the values of HTTP headers.                                                                                                                                                                                                                                                  |
+| CCC-050333  | Traffic mirroring                                     | Can duplicate incoming network traffic and send it to a secondary destination for monitoring, analysis, or testing purposes.                                                                                                                                                                                                                  |
+| CCC-050334  | Traffic splitting / weighted routing                  | Can distribute incoming traffic across multiple backend resources based on predefined weights or percentages. (e.g. for canary deployments, A/B testing, blue-green deployments, or gradual traffic migrations)                                                                                                                               |
+| CCC-050335  | Outpost support                                       | Ability to extend cloud-provisioned load balancing services to on-premises or edge locations using dedicated hardware or software appliances (outposts).                                                                                                                                                                                      |
```

Comment:
Looks vendor specific

Reply:
Thats works for me @cfisher-scottlogic

Reply:
Sorted! Also changed description slightly: [ea952e0](https://github.com/finos/common-cloud-controls/pull/149/commits/ea952e0f4b59cc62932bc59071017c3069b91cf6).